docs/publishers/config.rst

```diff
@@ -385,6 +385,37 @@ loads.
   where annotations are located on the page relative to the current scroll
   position.
 
+.. option:: focus
+
+  .. warning::
+
+      This is an experimental API and may change in future.
+
+  ``Object``. A structured object that defines a focused filter set for the available 
+  annotations on a page. When this object is passed to the config, the sidebar will add 
+  a UI button element that the user can toggle on or off to apply the filtered set of 
+  annotations defined by this ``focus`` object. This structure may define a particular 
+  ``user`` to focus on. Currently, only the ``user`` type is supported, but others may 
+  be added later.
+
+  .. note::
+    The focus ``user`` may not necessarily be the same user viewing the sidebar. 
+
+  For example:
+
+  .. code-block:: javascript
+    
+    window.hypothesisConfig = function () {
+      return {
+        focus: {
+          user: {
+            username: "foobar1234",
```

Comment:
Is there a way to indicate that `username` is required but the other two props are optional?